packages/devextreme-schematics/src/utility/patch.ts

```diff
@@ -0,0 +1,14 @@
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+
+export class CustomNodePackageInstallTask extends NodePackageInstallTask {
+    toConfiguration = (): any => {
+        const rootOptions = super.toConfiguration();
+        const resultObject = {
+            name: rootOptions.name,
+            dependencies: rootOptions.dependencies,
```

Comment:
Do rootOptions have dependencies property? I don't see it here https://github.com/angular/angular-cli/blob/main/packages/angular_devkit/schematics/tasks/package-manager/install-task.ts#L56